### ISSUE TITLE
Allow overriding height and width attributes

### ISSUE THREAD

**Issue by docwhat** [zabute/gatsby-plugin-svgr#8]
I'm using the config options of `{ icon: true, viewBox: false }`.

When doing (roughly) this:
```jsx

import CommentIcon from '../icons/comment-alt.svg'

export const Example = props => (
  <div>
    <CommentIcon width="1.125em" /> 
  </div>
)
```

The final render is (roughly):

```html
<div><svg viewBox="0 0 576 512" width="1em" height="1em">...</path></svg></div>
```

I expected the `width` to be `1.125em`.

**Comment by docwhat** [comment]
Note: Setting CSS does work around it, but I'd rather have the mark up just be correct out-of-the-box.

**Comment by zabute** [comment]
Looks like this is an upstream issue (tested it in a non-gatsby project just to be sure). It also doesn't seem to respect the ```viewBox: false``` option as [specified](https://github.com/smooth-code/svgr#viewbox) in the svgr docs.

@neoziro any thoughts on this?

**Comment by neoziro** [comment]
I think this is because options are passed using query string. Is it possible to provide options using an object instead?

**Comment by zabute** [comment]
Thanks @neoziro. Unfortunately,  [webpack-configurator](https://github.com/lewie9021/webpack-configurator)  only supports ```query```.

Adding options in the loader config throws:
```console
There were errors with your webpack config:
[1]
module.loaders.7.options
object.allowUnknown , "options" is not allowed
``` 
Gatsby v2 no longer depends on webpack-configurator, but until then (which could be a while) we can't use ```options```. Is there any way SVGR could handle options passed in via query?

**Comment by neoziro** [comment]
Is it possible to pass `false` and `true` using query?

**Comment by zabute** [comment]
I think yes because this is what I get with the options: ```js { ref: true, icon: true, viewBox: false }```

```js
 return React.createElement("svg", _extends({
    fill: "none",
    stroke: "currentColor",
    strokeWidth: 2,
    strokeLinecap: "round",
    strokeLinejoin: "round",
    className: "feather feather-search"
  }, props, {
    width: "1em",
    height: "1em",
    ref: svgRef
  }), _ref2, _ref3);
};
```

Everything is okay. I was also mistaken when  I said it wasn't respecting ```viewBox: false``` because its been removed.

Also, when I do

```jsx
<SearchIcon viewBox="0 0 12 12" width="4em" height="4em" />
```
I get

```html
<svg viewBox="0 0 12 12" width="1em" height="1em">
```

Other options like ```{ dimensions: false }``` are also handled properly. So its definitely a problem with ```width``` and ```height```

**Comment by zabute** [comment]
I'm taking a look at SVR's source to see if I can track it down.

**Comment by zabute** [comment]
After further investigation, I believe this is a bug with how svgr does the transforms. I edited the ```webpack.test.js``` to use query strings instead of objects and it seems to be getting the options correctly.

Even with the cli, ```--no-icon``` flag will result in the following component:
```jsx
 <svg viewBox="0 0 88 88" {...props} width="1em" height="1em">
```
The width and height attrs are inserted after the rest props are spread which is why we can't override it.

**Comment by zabute** [comment]
Resolved in svgr 1.9.1.

Thanks @neoziro
